src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,23 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * Returns true if {@code s} represents an unsigned integer
+     * e.g. 0, 1, 2, ..., {@code Integer.MAX_VALUE} <br>
+     * Will return false for any other non-null string input
+     * e.g. empty string, "-1", "+1", and " 2 " (untrimmed), "3 0" (contains whitespace), "1 a" (contains letters)
+     * @throws NullPointerException if {@code s} is null.
+     */
+    public static boolean isUnsignedInteger(String s) {
+        requireNonNull(s);
+
+        try {
+            int value = Integer.parseInt(s);
+            // "-0" and "+0" is successfully parsed by Integer#parseInt(String)
+            return value >= 0 && !s.startsWith("+") && !s.startsWith("-");
+        } catch (NumberFormatException nfe) {
+            return false;
+        }
```

Comment:
Good job on ensuring the in put is a valid unsigned integer and not a signed one.